src/common/Input/Input.jsx

```diff
@@ -1,14 +1,40 @@
-const Input = ({ name, value, labelText, placeholderText, onChange }) => {
+import React from 'react';
+import styles from './Input.module.css';
+
+const Input = ({
+  labelText,
+  placeholderText,
+  inputType,
+  inputValue,
+  inputName,
+  onChange,
+  isRequired,
+  minLength,
+}) => {
+  let required = false;
+
+  if (isRequired) required = true;
```

Comment:
Not really sure why do we need this extra `required` param, it would be enough just for `isRequired` prop